roles/default/kiali-deploy/templates/dashboards/envoy.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
     app.kubernetes.io/part-of: kiali
 spec:
   title: Envoy Metrics
-#  discoverOn: "envoy_server_uptime"
+  discoverOn: "envoy_server_uptime"
```

Comment:
I believe this needs an update too (the helm chart templates): https://github.com/kiali/helm-charts/blob/master/kiali-server/templates/dashboards/envoy.yaml#L11